Packs/Code42/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Code42",
     "description": "The Code42 INCYDR integration accelerates insider threat incident response and remediation procedures for potential data exfiltration across computers, email, cloud and SaaS apps.",
     "support": "xsoar",
```

Comment:
```suggestion
    "support": "partner",
```

Reply:
Add the following to Code42EventCollector.yml:
`supportlevelheader: xsoar`

This clarifies that the package itself is supported by Code42, while only the event collector is supported by XSOAR.